src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -459,8 +459,9 @@ public with sharing class CRLP_RollupUI_SVC {
 
         for (String field : fields) {
             Map<String, String> fieldObject = new Map<String, String>();
-            fieldObject.put('name', objectName+' '+field);
-            fieldObject.put('label', objectLabel +': '+UTIL_Describe.getFieldLabel(objectName,field));
+            //this method is reused for rollups, where obj concatenation is needed, and filter groups, where it is not
+            fieldObject.put('name', concatenate ? objectName+' ' : '' + field);
```

Comment:
@bethbrains  Can you add parenthesis around these operation to make it a little more readable:
`(concatenate ? objectName+' ' : '') + field`